corehq/apps/hqadmin/utils.py

```diff
@@ -0,0 +1,19 @@
+from dimagi.utils.logging import notify_exception
+from pillowtop.utils import get_all_pillows
+
+from .models import PillowCheckpointSeqStore
+
+
+def pillow_seq_store():
+    for pillow in get_all_pillows():
+        checkpoint = pillow.get_checkpoint()
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
+        if not created and checkpoint['seq'] < store.checkpoint_id:
+            notify_exception(
```

Comment:
Ah right sorry don't know what I was thinking with the string comparison. Sounds like it might be best to just remove this? Though even though they say not to use that first bit we do when we suspect a rewind